init.lua

```diff
@@ -521,3 +521,6 @@ function lib_mount.drive(entity, dtime, is_mob, moving_anim, stand_anim, jump_he
 
 	entity.v2 = v
 end
+
+-- print to log after mod was loaded successfully
+print ("[MOD] Library Mount loaded")
```

Comment:
I think `minetest.log` would be better, IMO.
```suggestion
minetest.log("info", "[MOD] Library Mount loaded")
```

Reply:
I am neutral on that but my PR was aiming at **compatibilty** with legacy clients and servers. If you scan the MT forum you will become aware that there seem to be quite many MT 0.4 servers around which are actively used by many players.

Reply:
Havings said that I would continue in using the one **load print** for better compatibilty with legacy MT software and would use the `minetest.log` for all other cases where applicable, if I may.